scripts/delete-federation.sh

```diff
@@ -23,6 +23,46 @@ set -o pipefail
 
 source "$(dirname "${BASH_SOURCE}")/util.sh"
 
+function delete-with-kubebuilder() {
```

Comment:
I'd like to rename this function as `delete-with-kubectl`

Reply:
How about delete-script-deployment?